models/fivetran_log__connector_status.sql

```diff
@@ -58,13 +63,23 @@ connector_metrics as (
         max(case when connector_log.event_subtype = 'sync_end' 
             then connector_log.created_at else null end) as last_sync_completed_at,
 
+        max(case when connector_log.event_subtype = 'status'
+                and {{ fivetran_utils.json_parse(string="connector_log.message_data", string_path=["status"]) }} ='SUCCESSFUL'
+            then connector_log.created_at else null end) as last_successful_sync_completed_at,
```

Comment:
new customers will have the important json status stuff in `sync_end` events, and it sounds like we'll be retroactively rolling that out to old customers in the future as well. i think the following could capture what we need for everyone? 
```suggestion
        max(case when connector_log.event_subtype in ('status', 'sync_end')
                and {{ fivetran_utils.json_parse(string="connector_log.message_data", string_path=["status"]) }} ='SUCCESSFUL'
            then connector_log.created_at else null end) as last_successful_sync_completed_at,
```